APISpecification.yaml

```diff
@@ -277,6 +292,27 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Variant'
+    VepSubmissionStatusResponse:
+      type: object
+      required:
+        - submission_id
+        - status
+      properties:
+        submission_id:
+          type: string
+        status:
+          type: string
+          description: >
+            The possible statuses for a VEP submission are based on the list of documented statuses of Seqera workflows
+            (see https://github.com/seqeralabs/tower-cli/blob/50dd04dd806acf8461dfc9b286faec7e874aec15/USAGE.md?plain=1#L254-L259),
+            except for the 'UNKNOWN' status, which is listed in Seqera docs without explanation how a workflow can get into that status.
+            If the tools api receives 'UNKNOWN' status from the Seqera platform, interpret it the same as FAILED.
+          enum:
+            - SUBMITTED
+            - RUNNING
+            - SUCCEEDED
+            - FAILED
+            - CANCELLED
     VepFormSubmissionCreatedResponse:
```

Comment:
What was our descision about error responses ? Do we include these from the start, not bother (assume that e.g. 404 and 500 are implemented for all endpoints and expected/handled by client) or wait for BE to add these later (after implementation?).

Reply:
Hmm, I should probably add a 404 here, because it would be different from how Seqera will respond.